firstvoices/backend/management/commands/convert_draftjs_to_html.py

```diff
@@ -2,11 +2,21 @@
 import logging
 
 from django.core.management.base import BaseCommand
+from draftjs_exporter.defaults import BLOCK_MAP, STYLE_MAP
 from draftjs_exporter.html import HTML
 
 from backend.models import Site, Song, Story, StoryPage
 from backend.models.widget import SiteWidget, WidgetSettings
 
+DRAFTJS_EXPORTER_CONFIG = {
+    "block_map": BLOCK_MAP,
+    "style_map": STYLE_MAP,
+    "entity_decorators": {
+        "FALLBACK": lambda props: props.get("children", ""),
```

Comment:
Fallback config strips out props["children"] which is the inner text wrapped by the entity